### PR TITLE
Add caddy trust as script and optimize entryscript

### PR DESCRIPTION
**Pull request by jochenmanz**
### 5. Checklist

- [ ] I have rebased my changes to remove merge conflicts
- [ ] I have written or adjusted the documentation according to my changes

### CODE REVIEW

**Comment by jinnoflife** [comment]
As we discussed via 1:1:
I get `error: attribute 'wait4x' missing` on `devenv up`. Also, the documentation for the new script (and also the old ones) is missing

**Review by jinnoflife — CHANGES_REQUESTED** [review]
As we discussed via 1:1:
I get `error: attribute 'wait4x' missing` during `devenv up`. Also, the documentation for the new script (and also the old ones) is missing

**Comment by jochenmanz** [comment]
@jinnoflife i tested the changes with the 23.11 and it was working for me. Can you retest?

File: docs/Home.md [hunk 1]
```diff
@@ -29,25 +29,33 @@ and needs to be adjusted when using a lower version.
...
 | key_buffer_size      | 16777216                                                                                           | Defines, how many index buffers can be held in memory                                                                       |
 | max_allowed_packet   | 134217728                                                                                          | Defines, how much data can be send within one request before the server blocks it.                                          |
 | table_open_cache     | 1024                                                                                               | Shopware uses many tables, This settings allows the server to keep more tables open                                         |
 | sql_mode             | STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION | the default from MySQL, but without ONLY_FULL_GROUP_BY                                                                      |
 
+# Built in Commands
```
Comment (jinnoflife) [inline docs/Home.md:51]:
```suggestion
# Built in commands
```

File: modules/scripts.nix [hunk 2]
```diff
@@ -138,5 +142,11 @@ in {
...
 
   # Processes
-  processes.entryscript.exec = "${entryScript}";
+  processes.entryscript.exec = ''
+    ${entryScript}
+  '';
+
+  scripts.caddy-trust.exec = ''
+    ${config.services.caddy.package}/bin/caddy trust
+  '';
```
Comment (jinnoflife) [inline modules/scripts.nix:151]:
```suggestion
  scripts.caddy-trust.exec = ''
    ${config.services.caddy.package}/bin/caddy trust
  '';

  # Processes
  processes.entryscript.exec = ''
    ${entryScript}
  '';
```

Comment (jinnoflife) [reply modules/scripts.nix:151]: You may also move `# Processes`  I just couldn't mark it

**Review by jinnoflife — CHANGES_REQUESTED** [review]

File: docs/Home.md (hunk 1 above)
Comment (jinnoflife) [reply docs/Home.md:51]: I'd also like to see an example of the execution of a command, e.g. `devenv shell caddy-trust`

File: modules/scripts.nix (hunk 2 above)
Comment (jochenmanz) [reply modules/scripts.nix]: The Process needs to be defined where the script is defined :) so i cannot move the process.

File: docs/Home.md (hunk 1 above)
Comment (jochenmanz) [reply docs/Home.md]: done

File: modules/scripts.nix (hunk 2 above)
Comment (jinnoflife) [reply modules/scripts.nix]: I meant below, so that the scripts are defined in a block 🙂

**Review by jinnoflife — APPROVED** [review]

**Pull request merged**